TestBenches/MatchProcessor_test.cpp

```diff
@@ -46,7 +47,7 @@ int main()
   const auto nAllStub = tb.nFiles(allStubPatternarray);
   vector<AllStubMemory<stubMemType>> allstub(nAllStub);
   const auto nVMStubs = tb.nFiles(vmStubPatternarray);
-  VMStubMEMemoryCM<stubMemType, 3, 3, kNMatchEngines> vmstub;
+  VMStubMEMemoryCM<vmStubMemType, kNbitsrzbinMP, kNbitsphibinMP, kNMatchEngines> vmstub;
```

Comment:
The new variables NbitsrzbinMP and kNbitsphibinMP are both 4 - the old code used template arguments that were 3. Why are these set to four here?

Reply:
It looks like the rebase erased some updates. I had modified this so the barrel was always 3, and just the disk R/Z used 4 bits. I'll push the fix soon.